section2/exercises/if-statements.js

```diff
@@ -44,6 +44,14 @@ Log to the console the appropriate age range
 var dogAge = 3;
 // Write your conditional here
 
+if (dogAge <= 2){
+  console.log('puppy');
+} else if (dogAge >= 3 && dogAge <= 9){
+  console.log('adult');
+} else if (dogAge >= 10){
+  console.log('elderly');
+}
+
 
```

Comment:
You could refactor this and make it a little more clear!

```javascript
if (dogAge <= 2) {
  console.log('puppy');
} else if (dogAge <= 9) {
  console.log('adult');
} else {
  console.log('elderly');
}
```

Only one of the statements will run each time, so after the first conditional we've eliminated any values less than or equal to 2. This only leaves values greater than 2, so you don't need to check for ages greater than or equal to 3. Same thing for the last one, we've eliminated all puppy and adult ages, so we don't need to check any age this time!

I want to stress that what you have works, but you could make it even cleaner.